dev-tools/reconfigurator-cli/tests/input/cmds-set-remove-mupdate-override.txt

```diff
@@ -0,0 +1,41 @@
+# Load example system with 7 sleds:
+#
+# sled 0: unset -> unset (unchanged)
+# sled 1: unset -> set
+# sled 2: set -> unset
+# sled 3: set -> set (unchanged)
+# sled 4: set -> set (changed)
+# sled 5: set -> set (unchanged) but change something else
+# sled 6: set -> sled removed
+#
+# We'll also add another sled below (new_sled_id) with
+# remove_mupdate_override set.
+#
+# We don't need any zones for this test, so disable that to keep the
+# outputs minimal.
+
+load-example --nsleds 7 --ndisks-per-sled 0 --no-zones
+
+# Set the field on sleds 2-6 (0-indexed).
+blueprint-edit latest set-remove-mupdate-override 9a867dc9-d505-427f-9eff-cdb1d4d9bd73 00000000-0000-0000-0000-000000000000
```

Comment:
Just out of curiosity - what was your workflow for figuring out what IDs correspond to which sleds as you wrote this test?